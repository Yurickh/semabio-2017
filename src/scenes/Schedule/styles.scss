@import "ambience/color";

.page-schedule {
  & > .week-picker > .day {
    color: $color-forest;
  }
}
