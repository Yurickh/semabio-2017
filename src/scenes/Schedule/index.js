import React, { Component } from 'react'

import Title from '../../components/Title'

import './styles.css'

class Schedule extends Component {
	render() {
		return (
			<section className="page-schedule">
				<Title>Cronograma</Title>

				<div className="week-picker">
					<div className="day -selected">SEG</div>
					<div className="day">TER</div>
					<div className="day">QUA</div>
					<div className="day">QUI</div>
					<div className="day">SEX</div>
					<div className="day">SAB</div>
					<div className="day">DOM</div>
				</div>
			</section>
		)
	}
}

export default Schedule
