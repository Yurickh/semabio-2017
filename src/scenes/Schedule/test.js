import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import Schedule from '../Schedule'

describe('Schedule', () => {
  it('should render correctly', () => {
    const tree = shallow(<Schedule />)
    expect(toJSON(tree)).toMatchSnapshot()
  })
})
