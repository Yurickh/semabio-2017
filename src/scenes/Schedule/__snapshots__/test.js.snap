// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schedule should render correctly 1`] = `<div />`;
